pulp_2to3_migration/app/tasks/migrate.py

```diff
@@ -28,18 +28,15 @@ def migrate_from_pulp2(migration_plan_pk, dry_run=False):
         migration_plan_pk (str): The migration plan PK.
         dry_run (bool): If True, nothing is migrated, only validation happens.
     """
+    # MongoDB connection initialization
+    connection.initialize()
+
     plan = MigrationPlan.objects.get(pk=migration_plan_pk)
+
```

Comment:
do we still want to keep dry-run mode? right now it does have any impact other than a logger message

Reply:
I think we can remove it for now and decide if we want it later.  It would be a struggle I think because the migrate steps work on the data created by the pre-migrate steps so if we're not saving any of that data, we can't know what the migrate steps are supposed to be copying.

We could have a dry-run mode that skips the mgiration steps and just logs what is pre-migrated.  But that can be discussed further I guess.